MIP/mip-n/README.md

```diff
@@ -0,0 +1,39 @@
+# MIP-n: Naming Conventions for Movement Protocol Design
+- **Description**: Naming conventions for actors in Movement protocol design, tests, and documentation.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Desiderata**: [MD-0](../MD/md-0)
```

Comment:
There is no MD accompanying this proposal. Please add it or remove this field.

Reply:
done.